src/Carousel/Carousel.tsx

```diff
@@ -51,26 +55,38 @@ const Carousel: RsRefForwardingComponent<'div', CarouselProps> = React.forwardRe
     const vertical = placement === 'left' || placement === 'right';
     const lengthKey = vertical ? 'height' : 'width';
 
-    const [activeIndex, setActiveIndex] = useState(0);
+    const [currentIndex, setCurrentIndex] = useState(activeIndex);
```

Comment:
Reference PanelGroup https://github.com/rsuite/rsuite/blob/main/src/PanelGroup/PanelGroup.tsx#L49
```suggestion
    const [activeIndex, setActiveIndex] = useControlled(activeIndexProp, defaultActiveIndex);
```